kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/SolverServicesClient.java

```diff
@@ -41,6 +42,17 @@ void solvePlanningProblem(String containerId,
     void terminateSolverEarly(String containerId,
                               String solverId);
 
+    void addProblemFactChange(String containerId,
+                              String solverId,
+                              ProblemFactChange problemFactChange);
+
+    void addProblemFactChanges(String containerId,
+                              String solverId,
+                              List<ProblemFactChange> problemFactChange);
```

Comment:
Here they are nicely typed to ProblemFactChange instead of Object. So I presume there's a reason for that - if it's not fixable in the Commands (I am intrested why in that case anyway), do add a comment that we except a ProblemFactChange instances there...